contents/blog/the-posthog-array-1-32-0.md

```diff
@@ -0,0 +1,133 @@
+2. We're removing support for insights with "Minute" intervals. From user feedback, these insights were hard to parse and could lead to significant performance issues in self-hosted instances. Please [reach out](/support) if you have any feedback on this. More details on the [PR](https://github.com/PostHog/posthog/pull/7847).
+
+### Talk to us about how we can improve
+
+We’re always working on improving the product experience and would love to talk to you! Please join one of our Product, Engineering, or Marketing team members on a quick 30-min call to help us understand how to improve. Schedule directly [on Calendly](https://calendly.com/posthog-feedback).
+
+As a small thank you for your time, we're giving away awesome [PostHog merch](https://merch.posthog.com)!
+
+
+### Experimentation launch 🚀
+
+We've been working hard on a brand new Experimentation feature which will let you quickly and confidently run experiments to test product improvements. This feature is currently available on Beta on PostHog Cloud, but if you'd like to be a beta tester or get a demo, please [reach out](https://posthog.com/slack) or schedule a [demo call](https://calendly.com/posthog-feedback) with the Engineering Team that created it.
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_32_0-experiments-sneak-peek.png" alt="Example screenshot: Experiments sneak peek" />
+
+## PostHog News
+
+Welcome to our new team members!
+
+
+[Andy](https://posthog.com/handbook/company/team#andy-vandervell-content-marketer) joined PostHog as our first Content Marketer. Andy is a definite 👎 on pineapple on pizza (🍍 on 🍕).
+
+> I am the only person called Andrew Vandervell in the universe.
+
+[Simon](https://posthog.com/handbook/company/team#simon-fisher-customer-success) joined us in Customer Success to help our Enterprise customers get the most value out of PostHog. Simon's stance on pineapple on pizza (🍍 on 🍕) is: On the fence.  I'd never actively order a pizza with pineapple on it, but as a pizza lover and pineapple was the only option - I’d still eat it.  Same response for Marmite.
+
+> I’m (loosely) related to an actor who’s starred in 6 Star Wars movies.
+
+
+## Community
+
+Want to help improve PostHog? We always welcome contributions from our community! Check out our [contributing resources](/docs/contribute) to get started.
+
+## Open roles
+
+Join us in helping make more products successful! We're currently hiring for the following roles:
+
+- Software Engineer
+- Full Stack Engineer
+- Tech Lead Manager/Engineering Manager
+- Community Engineer, more details here
+- Full Stack Engineer
```

Comment:
Full Stack Engineer twice (also, there's Software Engineers which seems redundant, I suppose there's more nuance to it in reality)